Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLaw.xml

```diff
@@ -0,0 +1,27 @@
+﻿<Document>
+  # Marine Law
+
+  ## Basics
+  - Marine law applies to all personnel in the vicinity of the operation or aboard the UNMC warship, including hostile factions and other organizations not explicitly excempt.
+  - Command and [color=#ff000d][bold]MP[/bold][/color] are required to both enforce the law and abide by it.
+  - Should [color=#ff000d][bold]MP[/bold][/color]s disagree on cases, Marine Law rank structure must be followed.
+  - [color=#ffa500][bold]Provost Marshal Office[/bold][/color] > [color=#00CBFE][bold]Commanding Officer[/bold][/color] > [color=#8B0000][bold]Chief MP[/bold][/color] > Warden > [color=#ff000d][bold]MP[/bold][/color]
+  - On UNMC warships and operations, the law is enforced by the [color=#ff000d][bold]MP[/bold][/color]s and [color=#8B0000][bold]Chief MP[/bold][/color], and they operate independently from the normal command structure ONLY in matters related to the enforcement of the Law.
+  - The [color=#00CBFE][bold]Commanding Officer[/bold][/color] of the warship has the final say on law enforcement within their operational area unless overseen by higher officials of the [color=#ffa500][bold]Provost Marshal Office[/bold][/color] or High Command.
+  - [color=#ff000d][bold]MP[/bold][/color] may, at their discretion, ignore Minor Crimes unless they are ordered to enforce one.
+  - [color=#ff000d][bold]MP[/bold][/color] cannot be ordered to ignore Minor Crimes.
```

Comment:
Apologies, I have no idea how github works so I'm not sure if I'm doing this right, but I noticed what seems to be a typo on this line. MP's may ignore crimes on line 11, but line 12 it says MP's cannot be ordered to ignore Minor Crimes. I think thats supposed to be Major crimes not minor.

Reply:
> Apologies, I have no idea how github works so I'm not sure if I'm doing this right, but I noticed what seems to be a typo on this line. MP's may ignore crimes on line 11, but line 12 it says MP's cannot be ordered to ignore Minor Crimes. I think thats supposed to be Major crimes not minor.

Intentional. MPs can choose to ignore a minor crime, but cannot be forced to by a superior. The wording was changed from "forced" to "ordered", which is confusing. I suggest using the word "compelled" to make it easier. If this is confusing to one person, it will likely be confusing to others.